build.yaml

```diff
@@ -427,9 +427,9 @@ common_cli:
 # flocker.node.functional is hanging, so we don't run it
 job_type:
   run_trial:
-  # http://build.clusterhq.com/builders/flocker-centos-7
+    # http://build.clusterhq.com/builders/flocker-centos-7
     run_trial_on_AWS_CentOS_7:
-      on_nodes_with_labels: 'aws-centos-7-T2Small'
+      on_nodes_with_labels: 'aws-centos-7-SELinux-T2Small'
```

Comment:
"because the labels must be differentiated from the last set of AMI labels"
that's pretty much it, the config in jenkins for nodes with 'aws-centos-7-T2small' use an AMI which doesn't contain SELinux enabled.

"should come up with a system (dates? version numbers?) for labelling"
Probably worth thinking of, question is if a new image for a slave should require changes to the build.yaml, in my view it shouldn't if one is simply updating the slave with OS patches.

On the other hand there are some benefits in doing so, for instance it would become visible in the build.yaml that the base slave image has changed  and  that any issues that occurred could be related back to that change